app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -45,6 +47,81 @@ func (cv *componentVersion) Type() string {
 	return "string"
 }
 
+// getVersionSet retrieves a set of available k8s API versions
```

Comment:
I'd link to [the code we took this from](https://github.com/helm/helm/blob/5cb9af4b1b271d11d7a97a71df3ac337dd94ad37/pkg/action/action.go#L310), people might not read the PR, assume it's our code and if a bug appears then a person will check the upstream and not try to fix it themselves